uint16 arb_id
uint8 sub_id
uint8[] data
std_msgs/Float64 t_stamp
